.eslintrc.yml

```diff
@@ -3,6 +3,7 @@
     ecmaVersion: 6
   env:
     browser: true
+    es6: true
```

Comment:
what's the difference between parseOptions ecmaVersion: 6 and es6: true? I would prefer if we just added specifically the options we needed.

Reply:
It's to nix the linter complaining about globals like `Map` not being defined. See <https://github.com/eslint/eslint/issues/5674>.

Reply:
considering we're doing es6 features on a piecemeal basis, I think it makes more sense if we specifically defined the globals we wanted to use?

Reply:
No longer relevant since #1663; will undo [feedback](https://github.com/new-xkit/XKit/pull/1414/commits/eb4b9ca3b521cd99bacbbe844204e6dad865458c#diff-0378d5111ab996d5882f4712e9350604) in rebase.